### PR TITLE
Fix for installing the DirectoryManger

### PR DESCRIPTION
**Pull request by Valdimus**
It was impossible to install the DirectoryManager. Some imports have changed from:
`from gevent.wsgi import WSGIServer `
to 
`from gevent.pywsgi import WSGIServer`

And it was impossible to install Flask if it was not at the format "Flask>=1.0.2" in the setup.py

I also update the requierements.txt and all test passed.

### CODE REVIEW

File: tests/test_storage_service.py [hunk 1]
```diff
@@ -139,6 +140,7 @@ def test_http(directory):
     # Test that service start
     http.start()
 
+    time.sleep(3)
```
Comment (Benvii) [inline tests/test_storage_service.py:143]: is it slower now ? ;)

File: setup.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
     description="Open Path View Directory Manager",
     long_description=open('README.md').read(),
     install_requires=[
-        "flask",
+        "Flask>=1.0.2",
```
Comment (Benvii) [inline setup.py:30]: Maybe we should also fix the version of other libraries to prevent futur changes ?

**Review by Benvii — APPROVED** [review]
Run test in containers got : 
9 passed, 2 warnings

Service correctly started in container (using systemctl).

Tested with an import (make-lot) on 214JJ_TEST, work well.

File: tests/test_storage_service.py (hunk 1 above)
Comment (Valdimus) [reply tests/test_storage_service.py:143]: Yes, but without it, the httpserver can't boot before we make the request.get... Maybe 1 second will do the trick.

**Pull request merged**